src/Umbraco.Web.UI.Client/src/common/resources/relation.resource.js

```diff
@@ -26,7 +26,7 @@ function relationResource($q, $http, umbRequestHelper) {
                     umbRequestHelper.getApiUrl(
                         "relationApiBaseUrl",
                         "GetByChildId",
-                        [{ childId: id, relationTypeAlias: alias }])),
+                        { childId: id, relationTypeAlias: alias })),
```

Comment:
I'm not too sure about the syntax @dawoe - why do we need to remove the `[` / `]`?

Reply:
Didn't mean to request changes, it was just a question ;-)

Reply:
If you pass it as an array the logic to turn it in to a querystring works different and parameters were not passed correctly to the api.

When you pass it as an object it works correctly.